setup.py

```diff
@@ -15,6 +15,7 @@
     install_requires=[
         "argcomplete",
         "docutils",
+        "dataclasses",
```

Comment:
Please don't hate me: "dataclasses" should come before "docutils" in alphabetical order